CredentialProvider.Microsoft/Util/EnvUtil.cs

```diff
@@ -35,6 +35,7 @@ public static class EnvUtil
         public const string MsalFileCacheEnvVar = "NUGET_CREDENTIALPROVIDER_MSAL_FILECACHE_ENABLED";
         public const string MsalFileCacheLocationEnvVar = "NUGET_CREDENTIALPROVIDER_MSAL_FILECACHE_LOCATION";
         public const string MsalAllowBrokerEnvVar = "NUGET_CREDENTIALPROVIDER_MSAL_ALLOW_BROKER";
+        public const string MsalBrokerWindowEnvVar = "NUGET_CREDENTIALPROVIDER_MSAL_BROKER_WINDOW";
```

Comment:
nit: can we rename this env varialble to ARTIFACTS_CREDENTIALPROVIDER - this way it will be clear its available to our other protocol credprovider wrappers such as the python https://github.com/microsoft/artifacts-keyring